styles/sections/_faqs.scss

```diff
@@ -0,0 +1,65 @@
+@use '../colors';
+
+.faq-list {
+  &:first-of-type {
+    @container body (min-width: 1024px) {
+      display: block;
+    }
+  }
+
+  li {
+    text-align: left;
+    border-bottom: 1px solid;
+    padding: 24px 0;
+
+    .h4,
+    h3 {
+      padding: 0 32px 0 0;
+      margin-bottom: 0;
+      text-transform: none;
+
+      @container body (min-width: 768px) {
+        padding-right: 48px;
+      }
+    }
+
+    .faq-details {
+      position: relative;
+
+      &[open] {
+        summary:before {
+          display: none;
+        }
+      }
```

Comment:
oooh cool idea to hide the `:before` when the details are open! I'm amazed no javascript was needed and this was able to be done with just html and css, i'll have to remember this for the future :-)

Reply:
Yeah same! I was kind of bummed though after we had just paired on the JS earlier yesterday afternoon and gotten it kind of working, but this was a lot more streamlined so seemed like the the better option.